server/server.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
```

Comment:
Why you are building another server ?
you have already the `App.js` or i'm missing something